apps/design/backend/src/ballots.ts

```diff
@@ -40,6 +41,43 @@ export function defaultBallotTemplate(state: string): BallotTemplateId {
   }
 }
 
+export function formatElectionForExport(
+  election: Election,
+  ballotStrings: UiStringsPackage,
+  precincts: Precinct[]
+): Election {
+  const splitPrecincts = precincts.filter((p) => hasSplits(p));
+
+  const signatureImageBySplit = splitPrecincts.flatMap((p) =>
+    p.splits.flatMap((split) =>
+      split.clerkSignatureImage
+        ? [[`${p.id}-${split.id}`, sha256(split.clerkSignatureImage)]]
+        : []
+    )
+  );
+  const sealOverrideBySplit = splitPrecincts.flatMap((p) =>
+    p.splits.flatMap((split) =>
+      split.electionSealOverride
+        ? [[`${p.id}-${split.id}`, sha256(split.electionSealOverride)]]
+        : []
+    )
+  );
```

Comment:
I think we should actually include all of the NH split override options, as all of these affect the rendered ballot:

https://github.com/votingworks/vxsuite/blob/6554fab5e4df912c76cbeb43cf5ef35588c4e71a/libs/hmpb/src/ballot_templates/nh_ballot_template.tsx#L58-L63

Reply:
Huh I thought these were already added to `hmpbStrings` but looks like I deprioritized merging https://github.com/votingworks/vxsuite/pull/5837.

I'll try to fold those changes into this PR

Reply:
Oh interesting, yeah missed that other PR!

Reply:
Called here: https://github.com/votingworks/vxsuite/pull/5991/files#diff-c268bf2996342639e8bfd885b518b620095551a7b47c8517f93bab768c57bfd0R150